specification/mcpp/resource-manager/Microsoft.CloudPartnerProgramMembership/preview/2023-02-09-preview/mcpp.json

```diff
@@ -0,0 +1,447 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-02-09-preview",
+    "title": "McppClient",
+    "description": "MCPP is the newest program set to track partners, measure their success, and qualify and reward them as Solution Partners across seven different designation groups.  The McppClient provides an interface to create an manage MCPP related purchases programatically"
+  },
+  "host": "management.azure.com",
+  "schemes": ["https"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "security": [
+    {
+      "azure_auth": ["user_impersonation"]
+    }
+  ],
+  "securityDefinitions": {
```

Comment:
>securityDefinitions [](http://example.com/codeflow?start=3&length=19)

pl check all the warnings showing up here and address them.. [https://github.com/Azure/azure-rest-api-specs/pull/22524/checks?check_run_id=11348535390](https://github.com/Azure/azure-rest-api-specs/pull/22524/checks?check_run_id=11348535390)